int32 room_point

---
int32 goal_point